### PR TITLE
Added pytorch sprint to sprints

### PR DESCRIPTION
**Pull request by pgryko**

### CODE REVIEW

File: data/pages-content/sprints.md [hunk 1]
```diff
@@ -207,6 +207,24 @@ Writing PEPs for the given Issues below. The organizer has never written [a PEP]
    - [Not Operator](https://github.com/python/typing/issues/801)
    - [copy_kwargs function](https://stackoverflow.com/questions/71968447/python-typing-copy-kwargs-from-one-function-to-another)
 
+### Pytorch - Tensors and Dynamic neural networks in Python
+
+- Number of people: ~10 (1 organizer)
+- Room: RB 104
```
Comment (rodrigogiraoserrao) [inline data/pages-content/sprints.md:213]: Please update the room occupancy list at the top of the sprint list.

**Review by rodrigogiraoserrao — CHANGES_REQUESTED** [review]
Hey there, thanks for your contribution!

Just one request before merging, annotated next to the issue.
While you're at it, can you please also merge `main` into your branch and make sure the pytorch sprint is at the bottom of the merged sprint list?

Thanks!

File: data/pages-content/sprints.md (hunk 1 above)
Comment (pgryko) [reply data/pages-content/sprints.md:213]: I've updated the room capacity

**Comment by pgryko** [comment]
> Hey there, thanks for your contribution!
> 
> Just one request before merging, annotated next to the issue. While you're at it, can you please also merge `main` into your branch and make sure the pytorch sprint is at the bottom of the merged sprint list?
> 
> Thanks!

Pytorch is a the bottom of the list. I'm sorry, I'm not sure what you mean regarding 'annotated next to the issue'.

**Comment by pgryko** [comment]
> > Hey there, thanks for your contribution!
> > Just one request before merging, annotated next to the issue. While you're at it, can you please also merge `main` into your branch and make sure the pytorch sprint is at the bottom of the merged sprint list?
> > Thanks!
> 
> Pytorch is a the bottom of the list. I'm sorry, I'm not sure what you mean regarding 'annotated next to the issue'.

Ah, I get it, you mean that you commented next to the issue. I've made the change and caught up with main

**Review by rodrigogiraoserrao — APPROVED** [review]
Thanks a lot! Looks good to me.

File: data/pages-content/sprints.md [hunk 2]
```diff
@@ -220,6 +220,24 @@ Writing PEPs for the given Issues below. The organizer has never written [a PEP]
...
+- Python Level: 2+ years of experience recommended and some C++ experience
+- Contact: Piotr Gryko ([Email](mailto:piotr.gryko@gmail.com), [GitHub](https://github.com/pgryko))
+  [Pytorch Forum](https://discuss.pytorch.org/)
+  [pytorch Github](https://github.com/pytorch/pytorch)
+- Links:
+  - [Building from source using docker (recommended)] (https://github.com/cresset-template/cresset)
+  - [Building from source] (https://github.com/pytorch/pytorch#from-source)
+  - [Contributing docs] (https://github.com/pytorch/pytorch/blob/main/CONTRIBUTING.md)
```
Comment (rodrigogiraoserrao) [inline data/pages-content/sprints.md:234]: Is the space between the text inside square brackets and the link inside parenthesis intentional?

**Review by rodrigogiraoserrao — COMMENTED** [review]
Just a final question about three of the links.

File: data/pages-content/sprints.md (hunk 2 above)
Comment (pgryko) [reply data/pages-content/sprints.md:234]: Nope - that's a defect. I'll fix it, catch up from main, confirm that the sprint is still on the bottom and merge the PR

Thank you!

Comment (rodrigogiraoserrao) [reply data/pages-content/sprints.md]: I went ahead and did it for you, @pgryko.
If you confirm there is nothing else _you_ would like to change, I'm happy to merge.

Comment (pgryko) [reply data/pages-content/sprints.md]: I'm happy with it, please go ahead and merge

**Pull request merged**